docs/microservices/logging-monitoring.md

```diff
@@ -59,11 +63,13 @@ Application Insights has a maximum throughput measured in events/second, and it
 
 A single operation may generate several telemetry events, so if the application experiences a high volume of traffic, it is likely to get throttled. To mitigate this problem, you can perform sampling to reduce the telemetry traffic. The tradeoff is that your metrics will be less precise. For more information, see [Sampling in Application Insights](/azure/application-insights/app-insights-sampling). You can also reduce the data volume by pre-aggregating metrics &mdash; that is, calculating statistical values such as average and standard deviation, and sending those values instead of the raw telemetry. The following blog post describes an approach to using Application Insights at scale: [Azure Monitoring and Analytics at Scale](/archive/blogs/azurecat/azure-monitoring-and-analytics-at-scale).
```

Comment:
`The tradeoff is that your metrics will be less precise.`
This is not true if the instrumentation supports pre-aggregation. When pre-aggregation is present, the main tradeoff is that you have fewer trace samples for edge case debugging and diagnostics.